extensions/positron-python/python_files/create_venv.py

```diff
@@ -89,9 +89,9 @@ def venv_exists(name: str) -> bool:
 def run_process(args: Sequence[str], error_message: str) -> None:
     try:
         print("Running: " + " ".join(args))
-        subprocess.run(args, cwd=os.getcwd(), check=True)
-    except subprocess.CalledProcessError:
-        raise VenvError(error_message)
+        subprocess.run(args, cwd=os.getcwd(), check=True)  # noqa: PTH109
```

Comment:
They added LOTS of ruff rules upstream, which I did not pull in since I'd rather have it be a follow up PR due to the amount of code changes (800+ errors that are not auto-fixable). I'll update the `pyproject.toml` to match upstream with the breaking rules commented out to pass CI.